### DISCUSSION TITLE
blog/2024/05/04/first-blog-post

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#26] [General]
# blog/2024/05/04/first-blog-post

趁着五一劳动节的时间，把这个网站做了出来，顺便学会了很多的前端react知识。

https://996.ninja/blog/2024/05/04/first-blog-post

**Comment by zzx856** [comment]
cool！！

**Reply by MingGH** [reply]
你是会挖坟的！🥸

**Comment by Shawyuan2022** [comment]
Ciallo～(∠・ω< )⌒★

**Reply by MingGH** [reply]
( *￣▽￣)((≧︶≦*)
